server/src/main/java/com/cloud/configuration/ConfigurationManagerImpl.java

```diff
@@ -410,6 +410,10 @@
     public final static ConfigKey<Long> IOPS_MAX_WRITE_LENGTH = new ConfigKey<Long>(Long.class, "vm.disk.iops.maximum.write.length", "Advanced", "0",
             "Maximum IOPS write burst duration (seconds). If '0' (zero) then does not check for maximum burst length.", true, ConfigKey.Scope.Global, null);
 
+    public static final ConfigKey<Boolean> AddHostOnServiceRestart = new ConfigKey<Boolean>(Boolean.class, "add.host.on.service.restart", "Advanced", "true",
```

Comment:
Considering that this is defined as a `static final`, I would recommend keeping the code convention such as described on [Oracle Java Naming Conventions](https://www.oracle.com/java/technologies/javase/codeconventions-namingconventions.html).

In this case `AddHostOnServiceRestart` should be changed to something as `ADD_HOST_ON_SERVICE_RESTART`.

Reply:
@GabrielBrascher I made the necessary changes. Please review again